test/config_SUITE.erl

```diff
@@ -1086,3 +1087,13 @@ config(_Config)  ->
     ?error_option(set_cfg_value([path_management, idle_timeout], invalid, ?PGW_PROXY_CONFIG)),
     ?error_option(set_cfg_value([path_management, down_timeout], invalid, ?PGW_PROXY_CONFIG)),
     ok.
+
+ergw_c_node_config() ->
+    [{doc, "Test for integrity of the configuration file ergw/config/ergw-c-node.config"}].
+ergw_c_node_config(_Config) ->
+    case file:consult("../../../../config/ergw-c-node.config") of
```

Comment:
if we do this (and I'm not yet sure that we should), shouldn't the file also pass a run through `ergw_config:validate_config/1` ?

Reply:
I add this test by reason that sometimes we can push incorrect config example, I suppose this test is needed for us. I don't use `ergw_config:validate_config.1` because here https://github.com/travelping/ergw/blob/master/src/ergw_config.erl#L260 we will throw options like `nodes`, `handlers` etc.